desktop/caliban.py

```diff
@@ -4684,14 +4687,25 @@ def save(self):
             self.raw and self.annotated are arrays to save in npz (self.raw should always
                 remain unmodified, but self.annotated may be modified)
         '''
+        # make sure has same dims as original
+        if self.dims == 3:
+            raw = np.squeeze(self.raw, axis=0)
+            ann = np.squeeze(self.raw, axis=0)
+        elif self.dims == 4:
+            raw = self.raw
+            ann = self.annotated
+        elif self.dims == 5:
+            raw = np.expand_dims(self.raw, axis=0)
+            ann = np.expand_dims(self.annotated, axis=0)
```

Comment:
I've seen this little pattern before. Would it make sense to have a helper method that casts to the right dimension?

Reply:
I'm not sure. I don't anticipate using this pattern very broadly so I think it would be fine to keep as is for now